packages/devtools/lib/src/timeline/timeline.dart

```diff
@@ -52,11 +47,21 @@ const Color slowFrameColor = Color(0xFFE50C0C);
 // Blue A700 is light, Indigo A400 is dark
 const Color selectedGpuColor =
     ThemedColor(Color(0xFF2962FF), Color(0xFF3D5AFE));
-// Dark Blue is light, Indigo A200 is dark
+// Dark Blue is light, Deep Purple A200 is dark
 const Color selectedCpuColor =
-    ThemedColor(Color(0xFF09007E), Color(0xFF536DFE));
-// Selection is high-contrast need white font.
-const Color hoverTextColor = ThemedColor(Colors.white, defaultForeground);
+    ThemedColor(Color(0xFF09007E), Color(0xFF7C4DFF));
+
+// Text color defaultForeground is too gray hard to read text, make it a little
+// whitter, not jarring white #FFFFFF.
```

Comment:
Nit on comment phrasing:
Text color [defaultForeground] is too gray, making it hard to read the text in dark theme. We should use a more white color for dark theme, but not jarring white #FFFFFF.

Reply:
Done